src/index.js

```diff
@@ -0,0 +1,130 @@
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    temperature: 40
+};
+
+const tempText = document.querySelector("#tempValue");
+const landscapeText = document.querySelector("#landscape");
+const cityText = document.querySelector("#cityNameInput");
+const skyText = document.querySelector("#sky");
+
+const changeSky = (sky) => {
+    if (sky === "sunny") {
+        skyText.innerHTML = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️"
+    } else if (sky === "cloudy") {
+        skyText.innerHTML = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+    } else if (sky === "rainy") {
+        skyText.innerHTML = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+    } else if (sky === "snowy") {
+        skyText.innerHTML = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+    }
+};
+
+const changeSkyValue = () => {
+    const skyValue = document.getElementById('skySelect').value;
+    changeSky(skyValue);
+};
+
+const increaseTemp = () => {
+    state.temperature += 1;
+    updateTempUi();
+};
+
+const updateTempUi = () => {
+    tempText.innerHTML = `${state.temperature}`;
+    if (state.temperature >= 80) {
+        landscapeText.innerHTML = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        tempText.style.color = "red";
+    } else if (state.temperature <= 79 && state.temperature > 69) {
+        landscapeText.innerHTML = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        tempText.style.color = "orange";
+    } else if (state.temperature <=69 && state.temperature > 59) {
+        landscapeText.innerHTML = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"; 
+        tempText.style.color = "yellow";       
+    } else if (state.temperature <=59 && state.temperature > 49) {
+        landscapeText.innerHTML = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        tempText.style.color = "green"; 
+    } else if (state.temperature <=49) {
+        landscapeText.innerHTML = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        tempText.style.color = "teal";
+    }  
+};
+
```

Comment:
Same comment here about using an object to store these values (see above in`changeSky()`). 

You could have an object like:
const tempAndLandscapeObject = {
    80: ["🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂", "red"],
    70: ["🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷", "orange"],
}
```
Then you could iterate over the object and check the keys with `state.temperature` and get the array that has the two values you need to set a new landscape and temperature color